--- sympy-1.14.0.orig/setup.py	2026-02-24 17:41:54.597760025 +0100
+++ sympy-1.14.0/setup.py	2026-02-24 17:42:01.079169976 +0100
@@ -321,7 +321,7 @@
           },
           # Set upper bound when making the release branch.
           install_requires=[
-              'mpmath >= 1.1.0, < 1.4',
+              'mpmath >= 1.1.0, < 1.5',
           ],
           py_modules=['isympy'],
           packages=['sympy'] + modules + tests,
